src/content/blog/2023-09-04-tus-node-server-v1.0.0.md

```diff
@@ -1,85 +1,95 @@
-them locally on disk, on Google Cloud Storage or on AWS S3 (or any other
+protocol. It is capable of accepting uploads of all sorts and sizes, and storing
+them locally on disk, or remotely on Google Cloud Storage or AWS S3 (or any other
 S3-compatible storage system). Due to its modularization and extensibility,
 support for nearly any other cloud provider could easily be added.
 
 ## History
 
 The Node.js tus server, [tus-node-server][], was originally published by
 [@bhstahl][] in May 2016. As with many open source projects, not everyone has
-the time to maintain things for free in their own time and as such
+the time to maintain things for free in their own time and as such,
 [tus-node-server][] was more or less abandoned at the end of 2018.
 
-In 2021, [@acconut][], [@mitjab][], and others sprung some live into the server
-again by upgrading dependencies adding some missing essential features. However
-there was still a lot of work ahead before the server could be considered stable
-and on par with the [protocol specification][].
+In 2021, [@acconut][], [@mitjab][], and others sprung some life into the server
+again by upgrading dependencies and adding a few essential features that were
+missing. However, there was still a lot of work ahead before the server could
+be considered stable and on par with the [protocol specification][].
 
 At the start of 2023, [Transloadit][] graciously allowed me ([@murderlon][]) to
-work on the server while getting paid. This, together with the help of
+work on the server on company time. This, together with the help of
 contributors, kick-started the effort towards `1.0.0`.
 
 ## The new major release
 
-After a long beta phase the new major is now ready.
+After a long beta phase, the new major is now ready.
 
-Here is an overview of the features and fixes since work was resumed in 2023 up
-until the `1.0.0` release.
+Here is an overview of the features and fixes that were implemented since work was
+resumed in 2023, up until today's `1.0.0` release.
 
 ### Split packages under the `@tus` scope
 
-For modularity and to signal the official packages; the server has been split
-into separate packages under the `@tus` scope.
+For modularity and to signal the official packages, the server has been split
+into separate packages under the `@tus` scope:
 
-- [`@tus/server`][]. The tus server. Standalone or integrate it into your
+- [`@tus/server`][] The tus server. Standalone or integrate it into your
   Node.js server.
-- [`@tus/file-store`][]. Store files on disk.
-- [`@tus/s3-store`][]. Store files on AWS S3.
-- [`@tus/gcs-store`][]. Store files on Google Cloud Storage.
+- [`@tus/file-store`][] Store files on disk.
+- [`@tus/s3-store`][] Store files on AWS S3.
+- [`@tus/gcs-store`][] Store files on Google Cloud Storage.
 
 ### Highlights
 
 - 🔒 Completely rewritten in TypeScript
 - 📦 Split into separate packages
 - 🪝 New events and hooks system
```

Comment:
Maybe it is because I never update stuff, but this emoji is not rendering for me. I see a little box with a question mark.